apps/hash-graph/lib/graph/src/api/rest/utoipa_typedef/subgraph/edges.rs

```diff
@@ -134,7 +136,7 @@ impl Edges {
                         Entry::Occupied(entry) => {
                             entry.into_mut().insert(
                                 id.version(),
-                                edges
+                                edges,
```

Comment:
Not important, but any idea why this changed? Wouldn't a rustfmt check have failed a previous PR if this was a chance from it?

Reply:
I don't know, it seems that `rustfmt` does bail on this function. For consistency, I also added another `,` below in 3d9fa77